rosbag2_cpp/src/rosbag2_cpp/readers/sequential_reader.cpp

```diff
@@ -82,6 +82,10 @@ void SequentialReader::reset()
 void SequentialReader::open(
   const StorageOptions & storage_options, const ConverterOptions & converter_options)
 {
+  // TODO(karsten1987): Check if we can just storage the complete options struct
+  // Should these configs be serialized in the metadata somehow?
+  storage_config_uri_ = storage_options.storage_config_uri;
```

Comment:
Yes, I think we should store the options struct instead of individual members - I see no reason not to